app/helpers/application_helper.rb

```diff
@@ -202,10 +202,10 @@ def collection_link(collection, **kwargs)
     link_to body, collection.path, **kwargs
   end
 
-  def email_link(type = :default, text: nil, additional_info: nil)
-    # The allowed types for type is :default, :business, :privacy, and members.
-    # These options can be found in field :email_addresses of models/site_config.rb
-    email = SiteConfig.email_addresses[type] || SiteConfig.email_addresses[:default]
+  def email_link(type = :contact, text: nil, additional_info: nil)
```

Comment:
`email_link` is used throughout the website to generate contact emails, I think the default should be the contact email, not the `:default`. This also will make sure that template pages that use `email_link` will generate links for that. Contact, CoC, privacy, and TOC for example. Same goes for pages like the account settings, request account deletion and video page among others